### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by mmafe**
# Summary of Changes
- Adds a pull request template

### Is Content created by the story?
- [x] No
- [ ] If yes, the content is:
   - [ ] Accessible
   - [ ] Mobile-friendly
   - [ ] Version-controlled
   - [ ] Governed by appropriate roles and permissions
   - [ ] Organized by taxonomy

### Are there User-facing changes?
- [x] No
- [ ] If yes, then:
   - [ ] User documentation is updated
   - [ ] User training is updated

### Are there Upstream Code changes?
- [ ] No
- [x] If yes, then:
   - [x] Code is complete and commented.
   - [x] Code is secure to the best of our knowledge and does not contain anything that cannot be on a public repo (e.g. passwords)
   - Is a test necessary?
      - [x] No
      - [ ] If yes, then:
         - [ ] Add a test if existing tests do not effectively test the code change
         - [ ] Test passes
   - [x] Pull request (PR) is mergeable with *develop* branch
   - [x] PR has a meaningful title
   - [x] PR has a summary of changes
   - [ ] PR is peer reviewed and meets team standards.
   - [x] PR builds without error

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+### Are there Upstream Code changes?
+- [ ] No
+- [ ] If yes, then:
+   - [ ] Code is complete, commented, and mergeable against *develop* branch.
+   - [ ] Code is secure to the best of our knowledge and does not contain anything that cannot be on a public repo (e.g. passwords)
+   - Is a test necessary?
+      - [ ] No
+      - [ ] If yes, then:
+         - [ ] Add a test if existing tests do not effectively test the code change
+         - [ ] Test builds without error
```
Comment (brharp) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Does this mean "test passes"?

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+   - [ ] Pull request (PR) created to merge against *develop*
```
Comment (brharp) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Maybe change wording to "Pull request (PR) created to merge branch with *develop*"?

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+   - [ ] PR has a meaningful title
+   - [ ] PR has a summary of changes
+   - [ ] PR is peer reviewed and meets team standards.
+   - [ ] PR builds without error
+   - [ ] PR merged with *develop*
```
Comment (brharp) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Is this a Catch-22? Can the branch be merged before all checks are passed?

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# Summary of Changes
+- Include a summary of changes made by this pull request
```
Comment (brharp) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Should this be marked as text to be replaced (e.g. with _emphasis_)?

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 1 above)
Comment (mmafe) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Yes - updated to read "Test passes" instead.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 2 above)
Comment (mmafe) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: It looks like I mention this twice, so I've updated the line that says "Code is complete, commented and mergeable against develop branch" to say "Code is complete and commented." Then I've updated the line above to say "Pull request (PR) is mergeable with develop branch."

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 3 above)
Comment (mmafe) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Yes - I've removed the "PR merged with develop" check.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 4 above)
Comment (mmafe) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Text has been emphasized. I've also added opening and closing angle brackets around it.

**Review by brharp — APPROVED** [review]

**Pull request merged**